### DISCUSSION TITLE
Flatten response using SELECT query

### DISCUSSION THREAD

**Discussion by madebydor** [supabase/supabase#3929] [Questions]
Hi, I'm currently getting the following response:

```json
{
    "name":"Apple",
    "teams_members":[
       {"user_id":{"full_name":"Johnathan Doe"}},
       {"user_id":{"full_name":"Bob Franklin"}}
    ]
}
```
But I'm trying to flatten this response to this:
```json
{
    "name":"Apple",
    "teams_members":[
       {"full_name":"Johnathan Doe"},
       {"full_name":"Bob Franklin"}
    ]
}
```

I've been scratching my head the entire day without success. I'm new to Postgress.

Here is how my tables look like:

| members |         
|----------|    
| id  (uuid)            | 
| team_id (uuid references teams)            | 
|full_name  (text)              |

|teams|     
|----------|
|id  (uuid)             |
|name (text)              |

|teams_members|
|------------------|
|user_id (uuid references users) |
|team_id (uuid references teams) |

And this is my query:

````js
await supabase.from('teams').select(`
    name,
    teams_members(
        user_id(
            full_name
        )
    )
`)
.single()
````

Does it even make sense to have `teams_members`? I feel I'm over complicating things.

**Comment by steve-chavez** [comment]
Try with:

```js
await supabase.from('teams').select(`
    name,
    members(
        full_name
    )
`)
.single()
```

**Comment by bombillazo** [comment]
how can I flatten the data if I have fields coming from both `teams_members` and `members`??

**Reply by steve-chavez** [reply]
For that case you can use https://postgrest.org/en/stable/references/api/resource_embedding.html#spread-embedded-resource

**Reply by steve-chavez** [reply]
Check spreading related tables on https://supabase.com/blog/postgrest-11-prerelease

**Reply by bombillazo** [reply]
Thanks!

**Reply by cohlar** [reply]
The December 2022 pre-release blog article says "This pre-release is not deployed to Supabase cloud but you can try it out locally with the [Supabase CLI](https://supabase.com/docs/reference/cli/introduction)." Has this by any chance been deployed to Supabase cloud since then?

**Reply by steve-chavez** [reply]
@cohlar Yes, it's available now https://supabase.com/blog/postgrest-11-1-release

Will put a note on the pre-release blog post.

**Reply by cohlar** [reply]
Thanks.
Just adding that you need to do a pause/restore if on a free instance that was up more than a few days ago.

**Reply by alhxji** [reply]
Thank you for this @steve-chavez ! Thank you!

**Reply by w3yogesh** [reply]
> Check spreading related tables on https://supabase.com/blog/postgrest-11-prerelease

thanks
